src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -36,14 +37,15 @@ export default class ReviewCollapsibleChapter extends React.Component {
 
   focusOnPage(key) {
     const name = `${key.replace(/:/g, '\\:')}`;
-    // legend & label target array type form elements
-    focusOnChange(name, 'p, legend, label');
+    focusElement(document.querySelector(`div[data-chapter="${name}"] button`));
```

Comment:
When `handleEdit` gets called, we want the focus to go to the accordion button that says "Edit [CHAPTER_NAME]"